### PR TITLE
[WIP] Switch every network to discovery

### PR DESCRIPTION
**Pull request by dshulyak**
cc @adambabik

### CODE REVIEW

**Review by adambabik — APPROVED** [review]
LGTM

File: src/status_im/constants.cljs [hunk 1]
```diff
@@ -75,11 +76,11 @@
                            :UpstreamConfig {:Enabled true
                                             :URL     "https://ropsten.infura.io/z6GCTmjdP3FETEJmMBI4"}}}
    "rinkeby"     {:id     "rinkeby",
-                  :name   "Rinkeby",
+                  :name   "Rinkeby with discovery",
                   :config {:NetworkId (ethereum/chain-keyword->chain-id :rinkeby)
                            :DataDir   "/ethereum/rinkeby"}}
    "rinkeby_rpc" {:id     "rinkeby_rpc",
-                  :name   "Rinkeby with upstream RPC",
+                  :name   "Rinkeby with upstream RPC and discovery",
```
Comment (adambabik) [inline src/status_im/constants.cljs]: I wonder if it fits on a single line...

File: src/status_im/constants.cljs [hunk 2]
```diff
@@ -75,11 +76,11 @@
                            :UpstreamConfig {:Enabled true
                                             :URL     "https://ropsten.infura.io/z6GCTmjdP3FETEJmMBI4"}}}
    "rinkeby"     {:id     "rinkeby",
-                  :name   "Rinkeby",
+                  :name   "Rinkeby with discovery",
```
Comment (flexsurfer) [inline src/status_im/constants.cljs]: do we want to show this to users?

Comment (adambabik) [reply src/status_im/constants.cljs]: Why not? Eventually, we will have `Rinkeby` but for now, the idea is to increase awareness that Rinkeby is run in discovery mode.

Comment (jeluard) [reply src/status_im/constants.cljs]: @adambabik Sounds too low level. Also there is only one Rinkeby option so it sounds superfluous. Maybe we could document this in the wiki?

Comment (adambabik) [reply src/status_im/constants.cljs]: Not sure if anyone will check it there. We will get rid of it in two weeks I hope.

File: src/status_im/constants.cljs (hunk 1 above)
Comment (dshulyak) [reply src/status_im/constants.cljs]: @adambabik is it better to remove this option at all? similarly to what we agreed to do for rinkeby

Comment (adambabik) [reply src/status_im/constants.cljs]: I guess we agreed to remove options using LES. This one uses infura.

**Comment by dshulyak** [comment]
this change https://github.com/status-im/status-go/pull/929 needs to be merged before i can update status-go in this PR

**Pull request closed without merging**